app/assets/stylesheets/components/_card.scss

```diff
@@ -2,7 +2,11 @@
   background-color: $white;
   max-width: $container-skinny-width;
 
-  &-wide { max-width: 100%; }
+  &-wide {
+    max-width: 100%;
+    padding-left: 0;
+    padding-right: 0;
```

Comment:
We'll need to add some horizontal padding to all of the content on the profile so it's not flush with the edges, should that be done in a follow up PR?